src/set_user.c

```diff
@@ -556,7 +582,7 @@ _PU_HOOK
 	/* if set_user has been used to transition, enforce set_user GUCs */
 	if (curr_state != NULL && curr_state->userid != InvalidOid)
 	{
-		switch (nodeTag(parsetree))
+		switch (nodeTag((Node *) pstmt->utilityStmt))
```

Comment:
Hmm, why this change? And if `parsetree` in 12 version is effectively  `(Node*)pstmt->utilityStmt`, why aren't we removing whereever that was previously defined?

Reply:
N/m, I see it was in `compatibility.h` and got removed.